src/index.test.ts

```diff
@@ -182,6 +181,49 @@ describe("addKeypairToEnvFile", () => {
   });
 });
 
+describe("initializeKeypair", () => {
```

Comment:
Sweet let's get this is now. It looks great BTW, reloading and checking keypair etc is exactly how I'd do the tests.

Reply:
Can you update the README.md with a description, example etc. per the other items?

Reply:
Yep!